configs/config.pcengines_apu2_vboot

```diff
@@ -1,4 +1,4 @@
-CONFIG_LOCALVERSION="v4.9.0.7"
```

Comment:
@piotr-kleins do not use this config. Use the normal non-vboot configs. Vboot should not be enabled. CONFIG_VPD is working without vboot.